src/Testcontainers.Qdrant/Testcontainers.Qdrant.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Are all these really necessary? I would think the following would be enough:

```suggestion
        <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Reply:
Every other TestContainer module defines these target frameworks, so was sticking to the project convention. I've also added net462 for same reason Milvus module does: https://github.com/testcontainers/testcontainers-dotnet/pull/1131#discussion_r1507712752